Pipeline/pipeline.py

```diff
@@ -533,22 +608,25 @@ def q_q_plot(self):
             )
             plt.close()
 
-    def standardized_error_plot(self):
-        var = list(self.train.columns)  
-        erro = self.test - self.train
-        erro = erro / erro.std()
-        erro.dropna(inplace=True)
+    def plot_standardized_error(self):
+        """Plot and save the graphic for standardized error.

```

Comment:
Plot and automatically save is not good practice. 
Here we should follow the same format that we have for our plots in the ROSS main package, returning the figure to the user. With matplotlib we should return the axes (ax object), something like:
```python
def plot(x, y, ax=None):
    if ax is None:
        ax = plt.gca()
    ax.plot(x, y)
    return ax
```

Reply:
The figures are saved for a HTML file resuming all plots. Without it a pratical front-end isn't feasible, because it will be necessary to include a Java Script to make requests to .py file in order to generate the needed figures dynamically, which will be expensive to the browser. This HTML will be a resume of all results. For HTML or any front-end language, the best pratice is to create a folder with the needed images to generate a HTML page. We can also remove the front end, but I should not recommend it, because graphics itself could be misleading without proper context, which I created on HTML file to guide the user to correctly use pipeline and interpret the results.

Reply:
You can still save the figures for the HTML, it would just be in a different part of your code.
Something like:
```python
def generate_html():
    ...
    # save figures
    error_plot = self.plot_standardized_error()
    plt.savefig('error_plot.png')
```
Breaking this into separate parts of the code will allow the user to call the plot in other contexts in which they don't  want to save the plot.